ffi/src/logging.rs

```diff
@@ -0,0 +1,76 @@
+use std::fs::OpenOptions;
+use std::path::PathBuf;
+use std::sync::Once;
+
+use tracing::error;
+use tracing_subscriber::prelude::*;
+use tracing_subscriber::EnvFilter;
+
+static SETUP: Once = Once::new();
+
+const SSPI_DBG_LOG_PATH_ENV: &str = "SSPI_LOG_FILE_PATH";
+
+#[cfg(windows)]
+const HOME_PATH_ENV: &str = "USERPROFILE";
+
+#[cfg(not(windows))]
+const HOME_PATH_ENV: &str = "HOME";
+
+pub fn setup_logger() {
+    SETUP.call_once(|| {
+        let path = if let Ok(path) = std::env::var(SSPI_DBG_LOG_PATH_ENV) {
+            PathBuf::from(path)
+        } else if let Ok(path) = std::env::var(HOME_PATH_ENV) {
+            let mut path = PathBuf::from(path);
+            path.push("sspi-rs.log");
+            path
+        } else {
```

Comment:
In this case, we should probably not register any log subscriber if the `SSPI_LOG_PATH` env variable is not set. Or, do we want logs unconditionally? If we do, we probably don’t want to use user’s home directory by default anymore. This would be more appropriate I think: https://docs.rs/dirs/latest/dirs/fn.data_dir.html
But, as for me, I believe logs should be opt-in.

Input? @awakecoding @thenextman

Reply:
Yes, logs should definitely be opt-in

Reply:
I fixed it. Now `SSPI_LOG_PATH` has to be set for logging